lessons/Core/contracts/langs/en-us/index.adoc

```diff
@@ -342,7 +342,8 @@ It can be empowering for students to develop expertise on a topic and get to sha
 
 @lesson-instruction{
 - Let's do some more experimenting with some of the more complex image functions from the list we just saw! This time around we'll start with their contracts.
-- Complete @printable-exercise{triangle-contracts.adoc}, @printable-exercise{using-contracts.adoc} and @printable-exercise{radial-star.adoc} with your partner.
+- Complete @printable-exercise{triangle-contracts.adoc}, @printable-exercise{using-contracts.adoc} with your partner.
+- @opt{If time allows, complete @opt-printable-exercise{radial-star.adoc} with your partner.}
 }
 
```

Comment:
Confused why you want radial start to become less discoverable. Objections to adding it to the end of the opt directions that follow?